### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Banmahhhh**

### CODE REVIEW

File: rlschool/liftsim/environment/env.py [hunk 1]
```diff
@@ -75,7 +68,7 @@ def seed(self, seed=None):
     def step(self, action):
         time_consume, energy_consume, given_up_persons = self._mansion.run_mansion(action)
         reward = - (time_consume + 0.01 * energy_consume +
-                    1000 * given_up_persons) * 1.0e-5
```
Comment (TomorrowIsAnOtherDay) [inline rlschool/liftsim/environment/env.py]: This scale is expected to be 1e-3, as kejiao has told me.

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,78 @@
---index-url http://pip.baidu.com/root/baidu/+simple/
---trusted-host pip.baidu.com
--e .
+absl-py==0.7.1
```
Comment (TomorrowIsAnOtherDay) [inline requirements.txt]: Please remove abundant dependencies.

**Comment by TomorrowIsAnOtherDay** [comment]
@rical730 When can we have teamcity installed for this repository?

**Comment by rical730** [comment]
> @rical730 When can we have teamcity installed for this repository?

This afternoon

File: baseline/liftsim_baseline/rl_benchmark/agent.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                dtype='float32'
+            )
+            terminal = layers.data(name='terminal', shape=[], dtype='bool')
+            self._cost = self.alg.define_learn(
+                obs, action, reward, next_obs, terminal)
+    
+    def sample(self, obs):
+        exploration_ratio = 800000.0 / \
+            (800000.0 + self._global_step) + 0.1
+        if self._global_step > 200000 and self._global_step % 50000 <= 3000:
```
Comment (TomorrowIsAnOtherDay) [inline baseline/liftsim_baseline/rl_benchmark/agent.py]: I understand that this line is used for evaluation, but this function `def sample(self, obs)` is a function for exploration. I suggest that we should remove this line.

File: baseline/liftsim_baseline/wrapper_utils.py [hunk 4]
```diff
@@ -0,0 +1,176 @@
+import sys
+import random
+import numpy as np
+from rlschool.liftsim.environment.mansion.utils import ElevatorState, ElevatorAction
+from rlschool.liftsim.environment.mansion.utils import MansionAttribute, MansionState
+from rlschool.liftsim.environment.mansion.utils import EPSILON, HUGE
+from rlschool.liftsim.environment.mansion.mansion_config import MansionConfig
+from rlschool.liftsim.environment.mansion.mansion_manager import MansionManager
+
+
```
[the remark this answers is not in the document]
Comment (TomorrowIsAnOtherDay) [reply baseline/liftsim_baseline/wrapper_utils.py]: Function descriptions, arguments and returns are supposed to be detailed in comments.

File: rlschool/liftsim/environment/env.py [hunk 5]
```diff
@@ -63,7 +64,10 @@ def seed(self, seed=None):
         set_seed(seed)
 
     def step(self, action):
```
Comment (TomorrowIsAnOtherDay) [inline rlschool/liftsim/environment/env.py]: You should check that the length of `action` is equal to `self._mansion.attribute.ElevatorNumber * 2`
`
action = np.array(action)
assert(action.shape[0]) == self._mansion.attribute.ElevatorNumber * 2, 'action is supposed to be an array of length 1.
`

**Pull request merged**